sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/PerformAppServiceInteractionResponse.java

```diff
@@ -0,0 +1,56 @@
+package com.smartdevicelink.proxy.rpc;
+
+import android.support.annotation.NonNull;
+
+import com.smartdevicelink.protocol.enums.FunctionID;
+import com.smartdevicelink.proxy.RPCResponse;
+import com.smartdevicelink.proxy.rpc.enums.Result;
+
+import java.util.Hashtable;
+
+public class PerformAppServiceInteractionResponse extends RPCResponse {
+
+	public static final String KEY_SERVICE_SPECIFIC_RESULT = "serviceSpecificResult";
+
+	/**
+	 * Constructs a new PerformAppServiceInteractionResponse object
+	 */
+	public PerformAppServiceInteractionResponse() {
+		super(FunctionID.PERFORM_APP_SERVICES_INTERACTION.toString());
+	}
+
+	public PerformAppServiceInteractionResponse(Hashtable<String, Object> hash) {
+		super(hash);
+	}
+	/**
+	 * Constructs a new PerformAppServiceInteractionResponse object
+	 * @param success whether the request is successfully processed
+	 * @param resultCode whether the request is successfully processed
+	 */
+	public PerformAppServiceInteractionResponse(@NonNull Boolean success, @NonNull Result resultCode) {
```

Comment:
The constructor should have `serviceSpecificResult` as a param since it is mandatory